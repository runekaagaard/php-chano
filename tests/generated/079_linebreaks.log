
---- EXPECTED OUTPUT
<p>line 1</p>
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::linebreaks() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/079_linebreaks.php on line 4
---- FAILED
