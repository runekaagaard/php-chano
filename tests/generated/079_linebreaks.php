<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'line 1')));
foreach ($items as $i) echo $i->input->linebreaks();
