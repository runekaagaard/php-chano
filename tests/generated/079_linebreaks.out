Fatal error: Call to undefined method DtlIter::linebreaks() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/079_linebreaks.php on line 4